xml/Menu/mailchimptemplate.xml

```diff
@@ -1,9 +1,15 @@
 <?xml version="1.0"?>
 <menu>
-  <item>
-    <path>civicrm/mailchimptemplate</path>
-    <page_callback>CRM_Mailchimptemplate_Form_MailchimpTemplate</page_callback>
-    <title>MailchimpTemplate</title>
-    <access_arguments>access CiviMail</access_arguments>
-  </item>
+    <item>
+        <path>civicrm/mailchimptemplate</path>
+        <page_callback>CRM_Mailchimptemplate_Form_MailchimpTemplate</page_callback>
+        <title>MailchimpTemplate</title>
+        <access_arguments>access CiviMail</access_arguments>
+    </item>
+    <item>
+        <path>civicrm/mailchimptemplate/settings</path>
+        <page_callback>CRM_Mailchimptemplate_Form_MailchimpTemplateSettings</page_callback>
+        <title>MailchimpTemplateSettings</title>
```

Comment:
Maybe something nicer? Some whitespace could help

Reply:
It has 4 space indentation. Describe what do you mean "something nicer". It is nice enough for me:)

Reply:
I was thinking about the title :). Like: `Mailchimp Template Settings`

Reply:
Ah ok, I put a space to your title too:)